src/ray/core_worker/test/direct_task_transport_test.cc

```diff
@@ -178,15 +178,15 @@ class MockRayletClient : public WorkerLeaseInterface {
   }
 
   void RequestWorkerLease(
-      const TaskSpecification &resource_spec,
+      const TaskSpecification &resource_spec, bool grant_or_reject,
```

Comment:
Can we add a test to check that `grant_or_reject` is set properly by the client?